ApiReview/API.Polly.RateLimiting/NoDocs/Polly.RateLimiting/OnRateLimiterRejectedArguments.cs

```diff
@@ -0,0 +1,15 @@
+// Assembly 'Polly.RateLimiting'
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.RateLimiting;
+
+namespace Polly.RateLimiting;
+
+public readonly struct OnRateLimiterRejectedArguments
+{
+    public ResilienceContext Context { get; }
+    public RateLimitLease Lease { get; }
+    public TimeSpan? RetryAfter { get; }
```

Comment:
given that the value of `RetryAfter` is taken from the `Lease`, I believe we should remove the `RetryAfter` property from the arguments to make them lighter and avoid potential ambiguity like 'given that there are two ways to get it, which one should I use as a customer?"

Reply:
We can drop it.

Reply:
Addressed in #1506